build.sbt

```diff
@@ -361,7 +361,7 @@ lazy val protobufV3 = pekkoModule("protobuf-v3")
         .inProject),
     assembly / assemblyOption := (assembly / assemblyOption).value.withIncludeScala(false).withIncludeBin(true),
     autoScalaLibrary := false, // do not include scala dependency in pom
-    exportJars := true, // in dependent projects, use assembled and shaded jar
+    exportJars := false, // in dependent projects, use assembled and shaded jar
```

Comment:
IIRC,  osgi packaging require`exportJars = true` ?

Reply:
sbt-assembly 2.3.0 doesn't allow `exportJars := true` for this type of task- it fails